app/workers/reindex_alegre_workspace.rb

```diff
@@ -64,11 +67,14 @@ def get_request_docs_for_project_media(pm)
     end
   end
 
-  def check_for_write(running_bucket, event_id, team_id, write_remains=false, in_processes=3)
+  def check_for_write(running_bucket, event_id, team_id, write_remains=false, _in_processes=3)
```

Comment:
Q: Why do we need to keep the last argument `_in_processes`? I think we can remove it.